lib/sidekiq/superworker/processor.rb

```diff
@@ -1,14 +1,15 @@
+# frozen_string_literal: true
 module Sidekiq
   module Superworker
     class Processor
-      def complete(item, new_thread=true)
+      def complete(item, _new_thread = true)
         Superworker.debug "JID ##{item['jid']}: Sidekiq job complete"
 
         complete_item(item)
       end
 
-      def error(worker, item, queue, exception)
-        raise "Job has nil jid: #{item}" if item['jid'].nil?
+      def error(worker, item, _queue, exception)
+        fail "Job has nil jid: #{item}" if item['jid'].nil?
```

Comment:
this is minor, any reason for `fail` over `raise`?

Reply:
styling where raise is generally expected when you rescue vs. fail when you need to.